packages/twenty-server/src/engine/core-modules/file/file-upload/services/file-upload.service.ts

```diff
@@ -73,6 +81,7 @@ export class FileUploadService {
       filename: name,
       mimeType,
       fileFolder,
+      workspaceId,
     });
 
     return {
```

Comment:
incorrect path? It doesn't work on my side.
I also notice our implementation for ?token is flawed... there's a security hole. But that's still protected by the url's name. I guess that's for another issue!

Reply:
Yup, this had to be fixed. I didnt realize the path on the line 81 is then being used to access that file.